samples/createInstanceFromTemplateWithOverrides.js

```diff
@@ -62,13 +62,13 @@ function main(
       `Creating the ${instanceName} instance in ${zone} from template ${instanceTemplateName}...`
     );
 
-    // Receiving instance template
+    // Retrieve an instance template
```

Comment:
Retrieve an instance template by name.

---
Adding minor info about the process and ending with a period. Up to you to decide whether it's valuable to mention it gets the template by name or not.